de-lib/src/main/java/org/iplantc/de/commons/client/views/window/configs/ConfigFactory.java

```diff
@@ -118,8 +118,19 @@ public static TabularFileViewerWindowConfig newTabularFileViewerWindowConfig() {
         return ab.as();
     }
 
-    public static PathListWindowConfig newPathListWindowConfig() {
-        AutoBean<PathListWindowConfig> ab = applyWindowType(WindowType.DATA_VIEWER, factory.pathListWindowConfig());
+    public static HTPathListWindowConfig newHTPathListWindowConfig() {
+        AutoBean<HTPathListWindowConfig> ab = applyWindowType(WindowType.DATA_VIEWER, factory.pathListWindowConfig());
+        ab.as().setContentType(MimeType.PLAIN);
+        ab.as().setVizTabFirst(true);
+        ab.as().setSeparator(FileEditorServiceFacade.COMMA_DELIMITER);
+        ab.as().setColumns(1);
+        applyTag("Path List-" + DateTimeFormat.getFormat(PredefinedFormat.DATE_TIME_FULL).format(new Date()),
+                 ab);
+        return ab.as();
+    }
```

Comment:
Since we have to call `ab.as()` so many times, maybe this could use a variable at the beginning of the method to represent `ab.as()`.